play/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# this file describes how to build tsuru play image
+# to run it:
+# 1 - tsuru-admin platform-add play -d https://raw.github.com/tsuru/basebuilder/master/play/Dockerfile
+
+from ubuntu:14.04
+run apt-get update
+run apt-get install wget -y --force-yes
+run wget http://github.com/tsuru/basebuilder/tarball/master -O basebuilder.tar.gz --no-check-certificate
+run mkdir /var/lib/tsuru
+run tar -xvf basebuilder.tar.gz -C /var/lib/tsuru --strip 1
+run cp /var/lib/tsuru/play/deploy /var/lib/tsuru
+run cp /var/lib/tsuru/base/start /var/lib/tsuru
+run chmod -R 777 /var/lib/tsuru
```

Comment:
Hi @cezarsa I'm afraid it's not possible to chmod the files and commit them, since I'm on a Windows-machine. Also chmod +x /var/lib/tsuru works for installing, but on deploy I got errors copying the Procfile from /home/application/current, since it's nice-to-have a editable Procfile on a Play-project for additional (alternative-)configuration.

You may try to chmod the scripts, but as stated above, I'm not sure it resolves all the issues.

Thanks in advance for your reply!